core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -320,6 +320,11 @@ trait AttributeComponent {
       }
     }
 
+    val caseSensitiveCollate = SimpleExpression.unary[Option[String], Option[String]] { (value, qb) => 
+      qb.expr(value)
+      qb.sqlBuilder += " collate utf8_bin"
```

Comment:
How did this perform while testing? I'm curious about the before/after `explain` plan of this query with this collate added. 

In case it helps, you can enable the raw SQL output of these Slick queries in the logs when running locally. Just uncomment the `slick.jdbc.JdbcBackend.statement` logger in `logback.xml` (around line 100)

Reply:
Explain results:

```
MySQL [rawls]> explain select `value_string`, count(1) from `WORKSPACE_ATTRIBUTE` where ((`namespace` = 'tag') and (`name` = 'tags')) and (`value_string` is not null) group by `value_string` order by count(1) desc, `value_string`;
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
| id | select_type | table               | type | possible_keys | key  | key_len | ref  | rows  | Extra                                        |
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
|  1 | SIMPLE      | WORKSPACE_ATTRIBUTE | ALL  | NULL          | NULL | NULL    | NULL | 16719 | Using where; Using temporary; Using filesort |
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
1 row in set (0.056 sec)

MySQL [rawls]> explain select `value_string` collate utf8_bin, count(1) from `WORKSPACE_ATTRIBUTE` where ((`namespace` = 'tag') and (`name` = 'tags')) and (`value_string` is not null) group by `value_string` collate utf8_bin order by count(1) desc, `value_string` collate utf8_bin;
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
| id | select_type | table               | type | possible_keys | key  | key_len | ref  | rows  | Extra                                        |
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
|  1 | SIMPLE      | WORKSPACE_ATTRIBUTE | ALL  | NULL          | NULL | NULL    | NULL | 16721 | Using where; Using temporary; Using filesort |
+----+-------------+---------------------+------+---------------+------+---------+------+-------+----------------------------------------------+
1 row in set (0.058 sec)
```